mysql/datadog_checks/mysql/data/conf.yaml.example

```
@@ -170,14 +170,14 @@ instances:
         #
         # galera_cluster: false
 
-        ## @param extra_status_metrics - boolean - optional - default: true
+        ## @param extra_status_metrics - boolean - optional - default: false
         ## Set to `false` to disable extra status metrics.
         ##
         ## See also the MySQL metrics listing: https://docs.datadoghq.com/integrations/mysql/#metrics
         #
         # extra_status_metrics: true
```

Should that be this instead?
```suggestion
        ## @param extra_status_metrics - boolean - optional - default: false
        ## Set to `true` to enable extra status metrics.
        ##
        ## See also the MySQL metrics listing: https://docs.datadoghq.com/integrations/mysql/#metrics
        #
        # extra_status_metrics: false
```

---

We usually have the example matching the default

---

The description change makes more sense, but I think it's better to keep the example as `true`, it is the only value that makes sense when someone use this option